File: 0816-design-hashset/0816-design-hashset.java
class MyHashSet {
    Boolean bool[];
    public MyHashSet() {
        bool = new Boolean[1000001];
    }
    
    public void add(int key) {
        bool[key] = true;
    }
    
    public void remove(int key) {
        bool[key] = false;
    }
    
    public boolean contains(int key) {
        if(bool[key] == null || !bool[key]) return false;
        return true;
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * boolean param_3 = obj.contains(key);
 */

File: 0816-design-hashset/0816-design-hashset.py
class MyHashSet:

    def __init__(self):
        self.boolArray = [False]*(10**6 +1)

    def add(self, key: int) -> None:
        self.boolArray[key] = True

    def remove(self, key: int) -> None:
        self.boolArray[key] = False

    def contains(self, key: int) -> bool:
        return self.boolArray[key]


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)